### PR TITLE
[MIG] account_multicurrency_revaluation: Migration to 12.0

### PR DESCRIPTION
**Pull request by goliveirab**

### CODE REVIEW

**Comment by hbto** [comment]
https://github.com/Vauxoo/account-closing/blob/11.0/oca_dependencies.txt

Branch for account-financial-tools (dev branch) must be included in here so that runbot instance in oca can be properly built.

@goliveirab

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+account-financial-tools git@github.com:vauxoo-dev/account-financial-tools.git 12.0-account_reversal-hbto
+account-closing
```
Comment (hbto) [inline oca_dependencies.txt]: This needs to point towards https://github.com/vauxoo-dev/account-financial-tools/commits/12.0-account_reversal-hbto

in order it to work properly as in https://git.vauxoo.com/vauxoo/enterprise-mx/merge_requests/468/diffs#03a80bdbacce644481ba04cd71187eab200677f9_3_4

and this CI can run smoothly
https://runbot.odoo-community.org/runbot/repo/github-com-oca-account-closing-89 on your PR.

Regards

**Review by hbto — CHANGES_REQUESTED** [review]

File: account_multicurrency_revaluation/model/account.py [hunk 2]
```diff
@@ -114,11 +114,16 @@ def compute_revaluations(self, revaluation_date):
...
+                account_id).user_type_id
             accounts.setdefault(account_id, {})
-            accounts[account_id].setdefault(currency_id, {})
-            accounts[account_id][currency_id].\
-                setdefault(partner_id, {})
-            accounts[account_id][currency_id][partner_id] = line
+            receive_pay = True if account_type.id == \
+                self.env.ref("account.data_account_type_receivable").id or \
+                account_type.id == self.env.ref(
+                    "account.data_account_type_payable").id else False
```
Comment (hbto) [inline account_multicurrency_revaluation/model/account.py]: @goliveirab  

Save 

`
rec_pay = [self.env.ref("account.data_account_type_receivable").id, self.env.ref("account.data_account_type_payable").id]
`

in a variable outside the scope of the for loop thus saving cpu cycles.
and than make a make a comparison like:

`receive_pay = True if account_type.id  in rec_pay` else False

Comment (goliveirab) [reply account_multicurrency_revaluation/model/account.py]: @hbto 

Is this what you expect? https://github.com/Vauxoo/account-closing/pull/2/commits/560c5c4abcd24dc59b39a8cd6e33dd743f49e0f5

**Comment by hbto** [comment]
yes it is
Quien suscribe,

*Hbto*

**Pull request merged**